cms/djangoapps/contentstore/views/item.py

```diff
@@ -781,10 +781,17 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         visibility_state = None
     published = modulestore().has_published_version(xblock) if not is_library_block else None
 
-    #instead of adding a new feature directly into xblock-info, we should add them into override_type.
-    override_type = {}
+    # defining the default state 'True' of delete, drag and add actions into xblock_actions dict.
+    xblock_actions = {'deletable': True, 'draggable': True, 'addable': True}
+    explanatory_message = None
     if getattr(xblock, "is_entrance_exam", None):
-        override_type['is_entrance_exam'] = xblock.is_entrance_exam
+        # Entrance exam section should not be deletable, draggable and not have 'New Subsection' button.
+        xblock_actions['deletable'] = xblock_actions['addable'] = xblock_actions['draggable'] = False
```

Comment:
"deletable" and "draggable" refer to the actual xblock, correct? "addable" refers to whether or not children can be added to the xblock. Please rename "addable" to make that clear-- perhaps "childAddable".